internal/migrations/0004_auto_20190328_1736.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.7 on 2019-03-28 16:36
+
+from django.db import migrations
```

Comment:
Could we rename this migration, so to make it easier to see what was changed in each migration? E.g. `0004_member_phone_number_field`

Reply:
Sure